structuredevent-service-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/mapper/EnvironmentUseCaseMapper.java

```diff
@@ -51,9 +54,9 @@
         return useCase;
     }
 
-    private UsageProto.CDPEnvironmentStatus.Value lastStepToUseCaseMapping(String flowState) {
+    private UsageProto.CDPEnvironmentStatus.Value lastStepToUseCaseMapping(String nextFlowState) {
         UsageProto.CDPEnvironmentStatus.Value useCase = UsageProto.CDPEnvironmentStatus.Value.UNSET;
-        switch (flowState) {
+        switch (nextFlowState) {
```

Comment:
Might be confusing because in such a case the "UMS_RESOURCE_DELETE_STARTED_STATE" will be written to EDH instead of ENV_DELETE_FINISHED_STATE.